algorithms/kernel/dtrees/forest/classification/df_classification_predict_dense_default_batch_container.h

```diff
@@ -54,27 +54,69 @@ BatchContainer<algorithmFPType, method, cpu>::~BatchContainer()
 template <typename algorithmFPType, Method method, CpuType cpu>
 services::Status BatchContainer<algorithmFPType, method, cpu>::compute()
 {
-    Input * input                           = static_cast<Input *>(_in);
+    const Input * const input                           = static_cast<Input *>(_in);
     classifier::prediction::Result * result = static_cast<classifier::prediction::Result *>(_res);
     const classifier::Parameter * par       = static_cast<classifier::Parameter *>(_par);
     decision_forest::classification::Model * m =
         static_cast<decision_forest::classification::Model *>(input->get(classifier::prediction::model).get());
 
-    NumericTable * a = static_cast<NumericTable *>(input->get(classifier::prediction::data).get());
-    NumericTable * r =
+    const NumericTable * const a = static_cast<NumericTable *>(input->get(classifier::prediction::data).get());
+    NumericTable * const r =
         ((par->resultsToEvaluate & classifier::ResultToComputeId::computeClassLabels) ? result->get(classifier::prediction::prediction).get() :
                                                                                         nullptr);
-    NumericTable * prob = ((par->resultsToEvaluate & classifier::ResultToComputeId::computeClassProbabilities) ?
+    NumericTable * const prob = ((par->resultsToEvaluate & classifier::ResultToComputeId::computeClassProbabilities) ?
                                result->get(classifier::prediction::probabilities).get() :
                                nullptr);
 
     daal::services::Environment::env & env = *_env;
 
+    const VotingMethod defaultVotingMethod = VotingMethod::unweighted;
```

Comment:
minor comment: as we can see from header file, 'weighted' is default.

Reply:
I mean that "unweighted" is default method for old interface.